ee/clickhouse/queries/test/test_stickiness.py

```diff
@@ -30,4 +35,66 @@ def _create_person(**kwargs):
 
 
 class TestClickhouseStickiness(ClickhouseTestMixin, stickiness_test_factory(ClickhouseStickiness, _create_event, _create_person, _create_action, get_earliest_timestamp)):  # type: ignore
-    pass
+    @snapshot_clickhouse_queries
+    def test_filter_by_group_properties(self):
```

Comment:
I would start taking advantage of the `journeys_for` [method](https://github.com/PostHog/posthog/blob/3a323f361dcacdebf4823269e3f4de6322dcd413/ee/clickhouse/test/test_journeys.py) that creates persons and events more obviously